plugins/media/wowza/lib/model/WowzaMediaServerNode.php

```diff
@@ -51,7 +51,7 @@ public function getLiveWebServiceName()
 		return WowzaMediaServerNode::WEB_SERVICE_LIVE;
 	}
 	
-	public function getPlaybackHost($protocol = 'http', $format = null)
+	public function getPlaybackHost($protocol = 'http', $format = null, $deliveryType = null)
```

Comment:
I don't think you need both the format and the deliveryType, you can get one from the other.